src/lib/child-process-promisified.ts

```diff
@@ -36,6 +36,7 @@ export async function spawnPromise(
   cmd: string,
   cmdArgs: ReadonlyArray<string>,
   cwd: string,
+  isInteractive = false,
```

Comment:
Would `inheritOutput` perhaps be better?

```suggestion
  inheritOutput = false,
```

Reply:
The reasoning behind my suggestion:

`isInteractive` is my attempt to abstract away from details of implementation and focus on behavior. if tomorrow node api will change `stdio` option and to achieve interactive process you would need some other combination of stdio values or even other name you wouldn't need to change the name of your flag for other consumers inside your codebase or maybe outside if you decide to export this api.

But other than that fine with me. Technically it inherits not only output but also input, so we could also rename it to `inheritStdio`.

But ultimately up to you.

Reply:
Good point. Let's just leave this as-is.